cub/test/catch2_test_launch_helper.h

```diff
@@ -40,11 +41,11 @@
 //!
 //! ```
 //! // Add PARAM to make CMake generate a test for both host and device launch:
-//! // %PARAM% TEST_CDP cdp 0:1
+//! // %PARAM% TEST_LAUNCH lid 0:1
```

Comment:
Should that go to 2 if we add the `#if TEST_LAUNCH == 2` below?

Reply:
I was hesitant to add it here, not to make graph capture covered by default. But I think it's a good idea to at least document it. I'll add a section below.